web/packages/teleport/src/Discover/SelectResource/SelectResource.tsx

```diff
@@ -194,6 +190,14 @@ export function SelectResource({ onSelect }: SelectResourceProps) {
                       onSelect(r);
                     }
                   },
+                  onKeyUp: (e: KeyboardEvent) => {
```

Comment:
I think now we have the same logic for the select handers (`onClick`, `onKeyUp`) written twice.
Maybe instead we could extract it to the shared handler:
```ts
                const select = () => {
                  if (r.hasAccess) {
                    setShowApp(true);
                    onSelect(r);
                  }
                };
                resourceCardProps = {
                  onClick: select,
                  onKeyUp: (e: KeyboardEvent) => e.key === 'Enter' && select(),
                  tabIndex: 0,
                  role: 'button',
                };
```
(same below)